src/patternfly/components/AboutModalBox/styles.scss

```diff
@@ -0,0 +1,267 @@
+@import "../../patternfly-utilities";
+
+.pf-c-about-modal-box {
+
+  // Component variables
+  // Modal uses a non-standard background color
+  --pf-c-about-modal-box--BackgroundColor: $pf-color-black-1000;
+  // This is a special one-off glow for the about modal
+  --pf-c-about-modal-box--BoxShadow: 0 0 100px 0 rgba(256, 256, 256, .4);
+  --pf-c-about-modal-box--ZIndex: var(--pf-global--ZIndex--2xl);
+  // Width and height values are determined by design
+  --pf-c-about-modal-box--MaxWidth: pf-size-prem(1232px);
+  // Height is optimized for the exact height desired
+  --pf-c-about-modal-box--Height: pf-size-prem(762px);
+  // MinHeight is based on all the spacers we know we have, plus some LineHeight to make some space for at least one line of content in each section
+  --pf-c-about-modal-box--MinHeight: calc(var(--pf-c-about-modal-box__brand--PaddingTop) + var(--pf-c-about-modal-box__brand--PaddingBottom) + var(--pf-c-about-modal-box__header--PaddingTop) + var(--pf-c-about-modal-box__header--PaddingBottom) + var(--pf-c-about-modal-box__content--PaddingTop) + var(--pf-c-about-modal-box__content--PaddingBottom) + var(--pf-c-about-modal-box__brand-image--Height) + (1rem * 6 * var(--pf-global--LineHeight--md)));
+  // MaxHeight ensures that the modal will not go off the screen and instead the content will scroll
+  --pf-c-about-modal-box--MaxHeight: calc(100vh - var(--pf-global--spacer--xl));
+
+  //Margin left and right make sure there is some space around the modal at the largest breakpoint
+  --pf-c-about-modal-box--MarginLeft: var(--pf-global--spacer--3xl);
+  --pf-c-about-modal-box--MarginRight: var(--pf-global--spacer--3xl);
+
+  // Shared padding values
+  --pf-c-about-modal-box--PaddingTop: var(--pf-global--spacer--2xl);
+  --pf-c-about-modal-box--PaddingRight: var(--pf-global--spacer--2xl);
+  --pf-c-about-modal-box--PaddingBottom: var(--pf-global--spacer--2xl);
+  --pf-c-about-modal-box--PaddingLeft: var(--pf-global--spacer--2xl);
+
+  @media screen and (min-width: $pf-global--breakpoint--sm) {
+    --pf-c-about-modal-box--PaddingTop: var(--pf-global--spacer--3xl);
+    --pf-c-about-modal-box--PaddingRight: var(--pf-global--spacer--3xl);
+    --pf-c-about-modal-box--PaddingBottom: var(--pf-global--spacer--3xl);
+    --pf-c-about-modal-box--PaddingLeft: var(--pf-global--spacer--3xl);
+  }
+
+  // Element variables
+
+  // Brand
+  --pf-c-about-modal-box__brand--PaddingTop: var(--pf-global--spacer--2xl);
+  --pf-c-about-modal-box__brand--PaddingRight: var(--pf-c-about-modal-box--PaddingRight);
```

Comment:
I think this is appropriate because the component defines a shared padding value that is to be used to pad  sub-components consistently. In addition, this value is changed on a media query so that all sub-components then take the new value consistently. (assuming visual design was on board with the change in padding?)

Reply:
ok, let's if you both feel this is correct, lets please make a comment to explain it